src/macros.h

```diff
@@ -18,6 +18,26 @@
 #ifndef BASE_MACROS_H
 #define BASE_MACROS_H
 
+ // Unix and Windows handle new lines differently, we handle this here.
+#ifdef _MSC_VER
+#define SYS_NEW_LINE "\r\n"
+#define SYS_DOUBLE_NEW_LINE "\r\n\r\n"
+#define SYS_SEPARATOR '\\'
+#define SYS_NL_SIZE 2
+#else
+#define SYS_NEW_LINE "\n"
+#define SYS_DOUBLE_NEW_LINE "\n\n"
+#define SYS_SEPARATOR '/'
+#define SYS_NL_SIZE 1
+#endif // PLATFORM_WINDOWS
+
+// The ubiquitous stringify macros for formatting strings.
+#ifndef STRINGIZE
+#define STRINGIZE_HELPER(str) #str
```

Comment:
These were used in the none working warning and info macros that were taken out, I would leave them out unless we discover we really need them as I can't see us having a use for them.